packages/firebase_auth/example/lib/main.dart

```diff
@@ -147,58 +41,30 @@ class _MyHomePageState extends State<MyHomePage> {
       body: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
-          MaterialButton(
-              child: const Text('Test signInAnonymously'),
-              onPressed: () {
-                setState(() {
-                  _message = _testSignInAnonymously();
-                });
-              }),
-          MaterialButton(
-              child: const Text('Test signInWithGoogle'),
-              onPressed: () {
-                setState(() {
-                  _message = _testSignInWithGoogle();
-                });
-              }),
-          MaterialButton(
-              child: const Text('Test verifyPhoneNumber'),
-              onPressed: () {
-                _testVerifyPhoneNumber();
-              }),
           Container(
-            margin: const EdgeInsets.only(
-              top: 8.0,
-              bottom: 8.0,
-              left: 16.0,
-              right: 16.0,
+            child: RaisedButton(
+              child: const Text('Test registration'),
+              onPressed: () => _pushPage(context, RegisterPage()),
             ),
-            child: TextField(
-              controller: _smsCodeController,
-              decoration: const InputDecoration(
-                hintText: 'SMS Code',
-              ),
+            padding: const EdgeInsets.all(16),
+            alignment: Alignment.center,
+          ),
+          Container(
+            child: RaisedButton(
+              child: const Text('Test signin/signout'),
+              onPressed: () => _pushPage(context, SigninPage()),
             ),
+            padding: const EdgeInsets.all(16),
+            alignment: Alignment.center,
           ),
-          MaterialButton(
-              child: const Text('Test signInWithPhoneNumber'),
-              onPressed: () {
-                if (_smsCodeController.text != null) {
-                  setState(() {
-                    _message =
-                        _testSignInWithPhoneNumber(_smsCodeController.text);
-                  });
-                }
-              }),
-          FutureBuilder<String>(
-              future: _message,
-              builder: (_, AsyncSnapshot<String> snapshot) {
-                return Text(snapshot.data ?? '',
-                    style:
-                        const TextStyle(color: Color.fromARGB(255, 0, 155, 0)));
-              }),
         ],
       ),
     );
   }
+
+  void _pushPage(BuildContext context, Widget page) {
```

Comment:
If this is only done in one place it doesn't really make sense to break it out into its own method.

Reply:
It's done in 2 places, both in the registration page and the sign in page

Reply:
OK, it's fine then